src/components/space/functionality/ActionsSelect.tsx

```diff
@@ -31,7 +31,13 @@ const ActionsSelect = (): JSX.Element | null => {
     setSelectedActionTypes(Array.isArray(value) ? value : [value]);
   };
   return (
-    <Stack direction="row" alignItems="center" flexGrow={1} flexShrink={0}>
+    <Stack
+      direction="row"
+      alignItems="center"
+      flexGrow={1}
+      flexShrink={0}
+      gap={1}
```

Comment:
Any particular reason why you use `gap` without the `useFlexGap` prop ? Would it work with `spacing` instead ? Curious as to why.